src/components/AccountCard.vue

```diff
@@ -152,28 +156,56 @@ export default defineComponent({
             }
         };
 
-        const loadResources = () => {
-            ram_used.value = fixDec(
-                Number(accountData.value.ram_usage) / KILO_UNIT.value,
-            );
+        const determineRamUnit = (size: number) => {
```

Comment:
```suggestion
        const determineUnit = (size: number) => {
```
method isn't specif to RAM (used for NET as well)